ICAFIX/PostFix.sh

```diff
@@ -409,7 +409,7 @@ main()
 
 	# For interpreted modes, make sure that matlab/octave have access to the functions they need
 	export FSL_MATLAB_PATH="${FSLDIR}/etc/matlab"
-	local ML_PATHS="addpath('${FSL_MATLAB_PATH}'); addpath('${MATLAB_GIFTI_LIB}'); addpath('${this_script_dir}/scripts');"
+	local ML_PATHS="addpath('${FSL_MATLAB_PATH}'); addpath('$HCPCIFTIRWDIR'); addpath('${MATLAB_GIFTI_LIB}'); addpath('${this_script_dir}/scripts');"
```

Comment:
Should probably name the MATLAB_GIFTI_LIB variable consistently with other functions.

Reply:
The other scripts in ICAFIX/ use `'${HCPPIPEDIR}/global/matlab'`, not a variable.  The folder does contain the gifti library, so it isn't entirely inaccurate.  What are you suggesting, specifically?

Reply:
I think Tim brown wrote that before he understood as much.  That code does not use gifti, it is what we reference in the other scripts as '${HCPPIPEDIR}/global/matlab'.